FR #966: Improving dataset metadata (#1464)
FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec: specification/metadata/schema/dataset_instance_id.md

> # Dataset Instance ID

Reviewer: Duplicate or?

> The Dataset Instance Id of the FOCUS dataset for which the schema and its data conform. This indicates which dataset instance the data artifact aligns with.

Reviewer: ```suggestion
The Dataset Instance ID is a unique identifier for the specific dataset instance provided by the data generator. It identifies the dataset instance that this schema and the corresponding dataset artifacts are aligned with.
```

> The Dataset Instance Id of the FOCUS dataset for which the schema and its data conform to. This indicates which dataset instance the data artifact aligns with.

Reviewer 2: ```suggestion
The Dataset Instance Id of the FOCUS dataset for which the schema and its data conform. This indicates which dataset instance the data artifact aligns with.
```

> Dataset MUST be provided in the metadata. Dataset MUST be of type String and MUST NOT contain null values.

Reviewer: ```suggestion
Dataset Instance ID MUST be provided in the schema metadata. Dataset Instance ID MUST be of type String and MUST NOT contain null values. Dataset Instance ID MUST be a unique identifier within a data generator.
```

> Dataset MUST be provided in the metadata. Dataset MUST be of type String and MUST NOT contain null values.

Reviewer 2: ```suggestion
Dataset Instance Id MUST be provided in the metadata. Dataset Instance Id MUST be of type String and MUST NOT contain null values.
```

Author: Resolved by irena's suggestion
